rezervo/providers/sats/helpers.py

```diff
@@ -0,0 +1,16 @@
+import json
+import re
+
+
+def retrive_sats_page_props(html_content: str):
+    data_pattern = r'<script data-props="true" type="application/json">(.*?)<\/script>'
```

Comment:
Shouldn't the `/` in `"application/json"` be escaped? (like in the script tag)

Reply:
https://github.com/mathiazom/rezervo/pull/43/commits/4ab071ef60185945e9c1fe7cd396e4174c4f9232 Yes, I forgot. Fixed